web-examples/src/main/java/io/vertx/example/web/mongo/Server.java

```diff
@@ -0,0 +1,64 @@
+package io.vertx.example.web.mongo;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.example.util.Runner;
+import io.vertx.ext.mongo.MongoClient;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.handler.*;
+
+/**
+ * This is an example application to showcase the usage of MongDB and Vert.x Web.
+ *
+ * In this application you will see the usage of:
+ *
+ *  * JADE templates
+ *  * Mongo Client
+ *  * Vert.x Web
+ *
+ * The application allows to list, create and delete mongo documents using a simple web interface.
+ *
+ * @author <a href="mailto:pmlopes@gmail.com>Paulo Lopes</a>
+ */
+public class Server extends AbstractVerticle {
+
+  // Convenience method so you can run it in your IDE
+  public static void main(String[] args) {
+    Runner.runExample(Server.class);
+  }
+
+  @Override
+  public void start() throws Exception {
+
+    // deploy an embedded mongo do we do not need to worry about not having a local mongo server running.
+    // in a production environment you would not do this
+    vertx.deployVerticle("maven:io.vertx:vertx-mongo-embedded-db:3.0.0-milestone6");
+
+    // Create a mongo client using all defaults (connect to localhost and default port) using the database name "demo".
+    final MongoClient mongo = MongoClient.createShared(vertx, config().put("db_name", "demo"));
+
+    // To simplify the development of the web components we use a Router to route all HTTP requests
+    // to organize our code in a reusable way.
+    final Router router = Router.router(vertx);
+
+    // Enable the body parser to we can get the form data and json documents in out context.
+    router.route().handler(BodyHandler.create());
+
+    // Entry point to the application, this will render a custom JADE template.
+    router.get("/").handler(Routes::index);
```

Comment:
Routes only contains a single handler so might be simpler just to add it direct in here (less classes). Also having the entire example in a single class makes it easier for codetrans to convert it to other languages automatically.